### ISSUE TITLE
508-defect-2 [COGNITION]: Non-selected or opted-into situations SHOULD include confirmation or not appear on review pages

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#12996]
## [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

I've found several Review screen statements that are not as clear as they could be if they included an answer or were removed from the Review screen accordions. Several examples include the following:

* The Veteran's Information definition list on the Review page should either remove the "I live on a US military base outside of the US" or add a confirmation that this is correct. It's confusing to have this information, but then provide a civilian address below it that's not a base address.
* Information about your former marriages, children, stepchildren, "This occurred outside the US" should be removed unless the event did occur outside the US. This could include marriages, divorces, births, adoptions, etc.
* Child status (adopted, stepchild, self-care)

I'm attaching a few screen shots to show some of these examples that could be improved by adding a confirmation answer or removing from the Review screen if they're not applicable.

/cc @peggygannon for content awareness

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Tasks
- [ ] Step through forms to see which ones still need this
- [ ] Apply appropriate attribute

## Acceptance Criteria

- [ ] As a user, I want to understand and have high confidence that the data I am submitting is correct and accurate

## Environment

* Enter https://staging.va.gov/view-change-dependents/add-remove-form-686c/ in browser
* Check two or three dependent situations, including adding and removing at least one dependent each
* Navigate to the **Review** page and open the accordions
* Verify some rows could be removed or improved with confirmation language

## WCAG or Vendor Guidance (optional)

* [Error Prevention (Legal, Financial, Data): Understanding SC 3.3.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-reversible.html)

## Screenshots or Trace Logs

![Screen Shot 2020-08-28 at 9 54 25 AM](https://user-images.githubusercontent.com/934879/91612420-01c52780-e943-11ea-8ca9-2721e30aeea1.png)

---
![Screen Shot 2020-08-28 at 9 56 59 AM](https://user-images.githubusercontent.com/934879/91612422-025dbe00-e943-11ea-9331-a776ef88b858.png)

## Definition of done

- [ ] Tag Jennifer on this issue and ask if she wants to close
- [ ] Review and acknowledge feedback
- [ ] Fix and/or document decisions made.
- [ ] Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by jason-gcio** [comment]
Should this be done at the platform level given that these changes will require significant edits to the existing Form System.  Have other teams resolved these issues?

**Comment by 1Copenut** [comment]
@jason-gcio I've passed this issue on to Carol and the design system team. I'll leave you as an assignee for now.

**Comment by micahchiang** [comment]
chp 36 has been fixed.

**Comment by jason-gcio** [comment]
686, Ch31, COE need this as well

**Comment by micahchiang** [comment]
@caw310 - I've unassigned your team from this, as the work is directly for our vsa-ebenefits team. Additionally, there is an attribute available at the platform level: `hideEmptyValueInReview: true,` that meets the need of this ticket so I don't think any further work is required by the design system team.
